docs/dataset/core-concepts/index.md

```diff
@@ -130,6 +130,10 @@ which provides a quick and standardized high level overview of which models perf
 For step-by-step instructions, take a look at the quickstart for
 [model comparison](../quickstart.md/#step-5-compare-models).
 
+!!! tip
+    Use the `Filter Untested Datapoints` option to narrow down your metrics to only include datapoints that all selected
```

Comment:
Should we call this `Filter to Intersection` (and below)? `Filter Untested Datapoints` only applies in contexts where there is a single model.

Reply:
Yeah - I think we started with "filter to intersection" and changed to this. We can rename to "use common datapoints"?